src/PostgREST/Config.hs

```diff
@@ -148,6 +148,7 @@ readOptions = do
         Nothing -> pure Nothing -- no audience in config file
         Just aud -> case preview stringOrUri (aud :: String) of
           Nothing -> fail "Invalid Jwt audience. Check your configuration."
+          (Just "") -> pure Nothing
```

Comment:
Maybe remove the unnecessary parentheses around `Just ""` here, wonder why hlint doesn't catch this.